Makefile

```diff
@@ -396,3 +397,127 @@ image-mnist-job-test-build: ## Build container image with the MNIST job.
+	kubectl delete clusterserviceversion $$CLUSTER_SERVICE_VERSION -n redhat-ods-operator
+	kubectl delete namespace redhat-ods-operator
+
+.PHONY: install-rhoai-operator
+install-rhoai-operator: ## Install RHOAI Operator
+	@echo -e "\n ==> Installing OpenShift AI Operator \n"
+	-kubectl create ns redhat-ods-operator
+	kubectl create -f contrib/configuration/rhoai/rhoai-operator-subscription.yaml
+	@echo Waiting for rhoai-operator Subscription to be ready
+	kubectl wait -n redhat-ods-operator subscription/rhods-operator --for=jsonpath='{.status.state}'=AtLatestKnown --timeout=180s
+	@echo -e "\n==> Creating default Data Science Cluster \n"
+	kubectl apply -f contrib/configuration/rhoai/default-dsci.yaml --server-side
+	kubectl apply -f contrib/configuration/rhoai/default-dsc.yaml --server-side
+
+.PHONY: delete-opendatahub-operator
+delete-opendatahub-operator: ## Delete OpenDataHub operator
+	@echo -e "\n==> Deleting OpenDataHub Operator \n"
+	-kubectl delete subscription opendatahub-operator -n openshift-operators
+	-export CLUSTER_SERVICE_VERSION=`kubectl get clusterserviceversion -n openshift-operators -l operators.coreos.com/opendatahub-operator.openshift-operators -o custom-columns=:metadata.name`; \
+	kubectl delete clusterserviceversion $$CLUSTER_SERVICE_VERSION -n openshift-operators
+	-kubectl delete namespace opendatahub
+
+.PHONY: install-opendatahub-operator
+install-opendatahub-operator: ## Install OpenDataHub operator
+	@echo -e "\n==> Installing OpenDataHub Operator \n"
+	-kubectl create ns opendatahub
+	kubectl create -f contrib/configuration/odh/opendatahub-operator-subscription.yaml
+	@echo Waiting for opendatahub-operator Subscription to be ready
+	kubectl wait -n openshift-operators subscription/opendatahub-operator --for=jsonpath='{.status.state}'=AtLatestKnown --timeout=180s
+	sleep 2
+	kubectl wait --for=condition=available deployment/opendatahub-operator-controller-manager -n openshift-operators --timeout=180s
+	kubectl apply -f contrib/configuration/odh/default-dsci.yaml --server-side
+	kubectl apply -f contrib/configuration/odh/default-dsc.yaml --server-side
+
+##@ GPU Support
+.PHONY: install-nfd-operator
+install-nfd-operator: ## Install NFD operator ( Node Feature Discovery )
+	@echo -e "\n==> Installing NFD Operator \n"
+	-kubectl create ns openshift-nfd
+	kubectl create -f contrib/configuration/nfd-operator-subscription.yaml
+	@echo -e "\n==> Creating default NodeFeatureDiscovery CR \n"
+	@while [[ -z $$(kubectl get customresourcedefinition nodefeaturediscoveries.nfd.openshift.io) ]]; do echo "."; sleep 10; done
+	@while [[ -z $$(kubectl get csv -n openshift-nfd --selector operators.coreos.com/nfd.openshift-nfd) ]]; do echo "."; sleep 10; done
+	kubectl get csv -n openshift-nfd --selector operators.coreos.com/nfd.openshift-nfd -ojsonpath={.items[0].metadata.annotations.alm-examples} | jq '.[] | select(.kind=="NodeFeatureDiscovery")' | kubectl apply -f -
```

Comment:
![image](https://github.com/project-codeflare/codeflare-operator/assets/73656840/35687399-39ed-4b5a-a02e-1f751f8aedb3)
Not sure what could cause this error coming from here.

Reply:
Not sure myself either. I will test again on my end